dev-util/scons/scons-3.0.5-r1.ebuild

```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
-PYTHON_COMPAT=( python{2_7,3_{5,6}} )
+PYTHON_COMPAT=( python{2_7,3_{5,6,7}} )
```

Comment:
So all the differences were due to missing deps, correct?

Reply:
For performing the tests with python_targets_python3_7 I required to to use `/etc/portage/profile/package.use.stable.mask`:
```
dev-lang/python-exec -python_targets_python3_7
# for scons tests with python 3.7:
dev-libs/libxml2 -python_targets_python3_7
dev-python/lxml -python_targets_python3_7
dev-python/cython -python_targets_python3_7
dev-python/certifi -python_targets_python3_7
dev-python/setuptools -python_targets_python3_7
dev-python/cssselect -python_targets_python3_7
```
and `/etc/portage/package.accept_keywords/scons`
```
dev-util/scons ~amd64
=dev-python/cssselect-1.0.3 ~amd64
```

The test result are similar both for python 3.6 and 3.7